### PR TITLE
add Authentication component- helps to refactor login as guest/facebook

### PR DESCRIPTION
**Pull request by maheshkurma**
Added Authentication Component- 
trying to move/refactor login as Guest/Facebook under Authentication Component.
passed FACEBOOK_APP_ID through env variable
authentication modal- added cose(X) button and update css

### CODE REVIEW

File: src/Modules/Authentication/FacebookLoginComponent.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const FacebookLoginComponent = ({onFbLogin} ) => {
+  const responseFacebook = async (response) => {
+    console.log(response);
+    onFbLogin(response);
+  }
+
+  return (
+    <div style={{width: '100%'}}>
+      <FacebookLogin
+        appId="876144606126554"
```
Comment (sarath-yerra) [inline src/Modules/Authentication/FacebookLoginComponent.js:13]: Can we use appId as an env variable

Comment (maheshkurma) [reply src/Modules/Authentication/FacebookLoginComponent.js]: updated

**Comment by sarath-yerra** [comment]
Changes looks good!

**Review by sarath-yerra — APPROVED** [review]

**Review by davidpickering — APPROVED** [review]
Approved by Sarath, reviewed with the team during standup.

**Pull request merged**